.travis.yml

```diff
@@ -11,18 +11,20 @@ services:
   - docker
 
 env:
-  - PIPENV_NOSPIN=true
+  - POETRY_VERSION=0.12.8
 
 install:
   - pip install -U docker-compose
-  - pip install "pipenv==2018.10.13"
-  - docker --version && docker-compose --version && pipenv --version
-  - pipenv --bare install --deploy --ignore-pipfile --dev
+  - curl -sSL "https://raw.githubusercontent.com/sdispater/poetry/${POETRY_VERSION}/get-poetry.py" | python
```

Comment:
Author of Poetry here :-)

You should always use the `master`'s version of the `get-poetry.py` script because it does not follow the same versioning as Poetry. And since you already export `POETRY_VERSION` the installer will automatically pick it up. So `curl -sSL "https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py" | python` should be enough.

Reply:
@sdispater thanks for your review! I appreciate that!

My initial idea was that I a little bit worried that sometimes things from `master` might be broken by mistake. Is it the case with `poetry`?

Reply:
The thing is the `get-poetry.py` script is not related to the versioning of Poetry and a bugfix will always be made on `master` without waiting for a Poetry release.

Now, the `get-poetry.py` script is pretty stable and the logic behind it will not change much so only expect bug fixes.

Reply:
Thanks! Replaced as you have recommended.